src/app/modules/group/components/member-list/member-list.component.html

```diff
@@ -77,7 +77,7 @@
         </span>
         <span class="filler"></span>
         <div class="summary-actions">
-          <span (click)="onRemove()" [ngClass]="{disabled: state !== 'ready' || selection.length === 0}">
+          <span (click)="onRemove()" [ngClass]="{disabled: (removalInProgress$ | async) || state !== 'ready' || selection.length === 0}">
```

Comment:
`[ngClass]="{disabled: ...}"` can be replaced by `[class.disabled] = ".."`, shorter/cleaner in this case.

Reply:
fixed